rts/at/bridge/src/main/java/org/jboss/narayana/rest/bridge/inbound/InboundBridge.java

```diff
@@ -101,10 +103,10 @@ public void start() {
         }
 
         try {
-            TransactionManager.transactionManager().resume(transaction);
+        	TransactionManager.transactionManager().resume(transaction);
```

Comment:
I would avoid tabs and favour of space characters instead making sure there are four of them (unless the existing file already makes extensive use of tabs which is not the case here). If you enable the checkstyle plugin or enable it in your IDE using the wildfly checkstyle rules (as described in the link I gave in my previous review) then these errors/warnings will be hightlighted.